### PR TITLE
Devs

### PR DESCRIPTION
**Pull request by Dzionek95**
Please check our GUI
- button nuke (should print NUKEE in console)
- each pane in grid pane should print cell pressed (if it's shoots grid pane) or ship placed (if it's ships grid pane)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Maven 3.0.1
 ## Running  
 
 ```
-To execute: mvn exec:java -Dexec.mainClass="com.javaAcademy.Battleship2.App"
+To execute: exec:java -Dexec.mainClass="com.java_academy.gui.Main"
```
Comment (agawda) [inline readme.md:18]: `mvn exec:`, also you need to `mvn clean install` project first to generate jar.

File: src/main/java/com/java_academy/gui/Main.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import javafx.stage.Stage;
+
+import java.io.File;
+import java.net.URI;
+
+public class Main extends Application {
+
+    @Override
+    public void start(Stage primaryStage) throws Exception{
+        URI url=new File("src/main/java/com/java_academy/gui/board.fxml").toURI();
```
Comment (agawda) [inline src/main/java/com/java_academy/gui/Main.java]: You could load this with 
`FXMLLoader loader = new FXMLLoader(getClass().getResource("/board.fxml"));`. 
You have to put `board.fxml` in `/src/main/resources` for this to work.

File: src/main/java/com/java_academy/gui/Main.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        FXMLLoader fxmlLoader = new FXMLLoader();
+        Parent root = fxmlLoader.load(url.toURL());
+        primaryStage.setTitle("Battleships");
+        primaryStage.setScene(new Scene(root, 1050, 700));
```
Comment (agawda) [inline src/main/java/com/java_academy/gui/Main.java]: Magic numbers and also you can set this in FXML file.

**Review by agawda — CHANGES_REQUESTED** [review]
Fix readme. :v

File: src/main/java/com/java_academy/gui/Main.java (hunk 2 above)
Comment (Oplia512) [reply src/main/java/com/java_academy/gui/Main.java]: Thank you

File: src/main/java/com/java_academy/gui/Main.java (hunk 3 above)
Comment (Oplia512) [reply src/main/java/com/java_academy/gui/Main.java]: Thanks again

**Pull request closed without merging**